fonts/fonts.css

```diff
@@ -0,0 +1,29 @@
+@font-face { 
+  font-family: 'montserrat-font';
+  src:
+    url('/fonts/bold/montserrat-bold-webfont.woff2') format('woff2'),
+    url('/fonts/bold/montserrat-bold-webfont.woff') format('woff');
+  font-display: swap;
+  font-weight: bold;
+  font-style: normal;
+ }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6088588)